docs/index.md

```diff
@@ -33,17 +33,14 @@ better alternative.
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
-
-- `domain` (String) Your Auth0 domain name. It can also be sourced from the `AUTH0_DOMAIN` environment variable.
```

Comment:
To note that we're finally fixing this as well, if we have a default set for an attribute, it can't be required.